.github/workflows/rust.yml

```
@@ -66,7 +66,8 @@ jobs:
           key: cargo-cache-benchmark-${{ hashFiles('datafusion/**/Cargo.toml', 'benchmarks/Cargo.toml', 'datafusion-cli/Cargo.toml') }}
 
       - name: Check workspace without default features
-        run: cargo check --no-default-features -p datafusion
+        run: cargo check --no-default-features -p datafusion &&
+             cargo check --tests --no-default-features -p datafusion-common
```

Could we please make this a separate name (to make it easier to see in the UI

Like:

```suggestion
        run: cargo check --no-default-features -p datafusion
        
      - name: Check datafusion-common  without default features
        run: cargo check --tests --no-default-features -p datafusion-common
```

---

No problem at all.